### PR TITLE
feat: add Telegram example

### PR DESCRIPTION
**Pull request by DiFronzo**

### CODE REVIEW

File: telegram/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+A simple Telegram Bot Command.
+
+## Tutorial
+
+- Follow the
+  [official Telegram guide](https://core.telegram.org/bots#3-how-do-i-create-a-bot)
+  for creating a Bot.
+- [![Deploy this example](https://deno.com/deno-deploy-button.svg)](https://dash.deno.com/new?url=https://raw.githubusercontent.com/denoland/deploy_examples/main/telegram/mod.ts&env=TOKEN,BOT_NAME)
+- Input TOKEN and BOT_NAME env variable field. The token value should be
+  available from the BotFather. The bot name you have set that ends with `_bot`
```
Comment (satyarohith) [inline telegram/README.md:12]:
```suggestion
  available from the BotFather. The bot name is the bot username that ends with either `_bot` or `Bot`.
```

File: telegram/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  or `Bot`.
+- Click on **Create** to create the project, then on **Deploy** to deploy the
+  script.
+- Grab the URL that's displayed under Domains in the Production Deployment card.
+- Go into a browser and paste the following url:
```
Comment (satyarohith) [inline telegram/README.md:17]:
```suggestion
- Visit the following URL (make sure to replace the template fields):
```

File: telegram/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+-
+  - `https://api.telegram.org/bot<YOUR_TOKEN>/setWebhook?url=<DOMAIN_NAME>/<YOUR_TOKEN>`
+- Next add a command to the bot by using a browser and paste the following url:
```
Comment (satyarohith) [inline telegram/README.md:20]:
```suggestion
- Add a command to the bot by visiting the following URL:
```

File: telegram/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+-
+  - `https://api.telegram.org/bot<YOUR_TOKEN>/setMyCommands?commands=[{"command":"ping","description":"Should return a 'pong' from the Bot."}]`
+- Now you can invite the bot to a Group Chat or just PM the bot with the
+  following command "/ping".
+-
+  - `https://api.telegram.org/bot<YOUR_TOKEN>/setWebhook?url=<DOMAIN_NAME>/<YOUR_TOKEN>`
```
Comment (satyarohith) [inline telegram/README.md:26]:
```suggestion
```

I think this is not necessary here. Do you have anything in mind?

File: telegram/mod.ts [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+      if (errors) {
+        console.error(errors.map((error) => error.message).join("\n"));
+        return json({ error: "couldn't create the message" }, {
+          status: 500,
+        });
+      }
+
+      // Returns the answer and set status code 201.
+      return json({ dataTelegram }, { status: 201 });
```
Comment (satyarohith) [inline telegram/mod.ts:80]: Is `201` required by Telegram for the messages to be posted? I couldn't find about it in the docs...

File: telegram/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# Telegram Bot Command
+
+A simple Telegram Bot Command.
+
+## Tutorial
+
+- Follow the
+  [official Telegram guide](https://core.telegram.org/bots#3-how-do-i-create-a-bot)
+  for creating a Bot.
+- [![Deploy this example](https://deno.com/deno-deploy-button.svg)](https://dash.deno.com/new?url=https://raw.githubusercontent.com/denoland/deploy_examples/main/telegram/mod.ts&env=TOKEN,BOT_NAME)
```
Comment (satyarohith) [inline telegram/README.md:10]:
```suggestion
- Deploy the Bot by clicking on this button: [![Deploy this example](https://deno.com/deno-deploy-button.svg)](https://dash.deno.com/new?url=https://raw.githubusercontent.com/denoland/deploy_examples/main/telegram/mod.ts&env=TOKEN,BOT_NAME)
```

File: telegram/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Telegram Bot Command
+
+A simple Telegram Bot Command.
+
+## Tutorial
+
+- Follow the
```
Comment (satyarohith) [inline telegram/README.md:7]: Maybe we should switch to using numbered lists as these are step-by-step instructions.
```suggestion
1. Follow the
```

**Review by satyarohith — CHANGES_REQUESTED** [review]
I created a bot using the instructions and it's working as expected. It's nice to have an example of Telegram Bot. 

Thanks for working on it, @DiFronzo!

File: telegram/mod.ts (hunk 5 above)
Comment (DiFronzo) [reply telegram/mod.ts:80]: Nah, it only need to be successful.

File: telegram/README.md (hunk 4 above)
Comment (DiFronzo) [reply telegram/README.md:26]: No, can be removed.

**Comment by satyarohith** [comment]
Hi @DiFronzo. :) The suggested changes should be considered before resolving the conversation. If you're new to GitHub/Open Source, this guide will help you: https://opensource.guide/how-to-contribute. Let me know if you need any further assistance.

**Comment by DiFronzo** [comment]
> Hi @DiFronzo. :) The suggested changes should be considered before resolving the conversation. If you're new to GitHub/Open Source, this guide will help you: https://opensource.guide/how-to-contribute. Let me know if you need any further assistance.

Sorry. I do agree with your changes 👍🏻  Thanks for guidance.

**Comment by satyarohith** [comment]
> > Hi @DiFronzo. :) The suggested changes should be considered before resolving the conversation. If you're new to GitHub/Open Source, this guide will help you: https://opensource.guide/how-to-contribute. Let me know if you need any further assistance.
> 
> Sorry. I do agree with your changes 👍🏻 Thanks for guidance.

No worries. Please commit the suggestions and we can land this on main. :)

**Comment by satyarohith** [comment]
Hi @DiFronzo. Can you check the "Allow edits from maintainers" option available on the right sidebar of your pull request? I would like to push a few README updates to your branch.

**Comment by DiFronzo** [comment]
Hi @satyarohith, the only option I see is "Allow edits and access to secrets by maintainers" that is already checked.

**Comment by satyarohith** [comment]
> Hi @satyarohith, the only option I see is "Allow edits and access to secrets by maintainers" that is already checked.

I'm possibly unable to push to your PR because of the branch being `main`. Can you run the below command to pick up my changes?

```
git fetch git@github.com:satyarohith/deploy_examples.git d0344d298d46069a6895c2686a4d34bbf4e6df18 && git cherry-pick d0344d298d46069a6895c2686a4d34bbf4e6df18
```

Also, please merge changes from `main` of denoland/deploy_examples to your PR to fix the lint errors.

**Review by satyarohith — APPROVED** [review]
LGTM.

Thanks for the contribution, @DiFronzo!

**Pull request merged**